index.d.ts

```diff
@@ -37,6 +37,7 @@ declare module 'plaid' {
 
   interface AssetReportCreateOptions {
     client_report_id?: string | null;
+    webhook?: string;
```

Comment:
`string | null` ?

Reply:
Nope, this one is actually just `string`.